chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestGroup.java

```diff
@@ -1,51 +1,36 @@
 package com.griddynamics.jagger.user.test.configurations;
 
+import java.util.Collections;
 import java.util.List;
 
 /**
  * Describes the group of {@link JTest} that should run in parallel.
  */
 public class JTestGroup {
-    private String id;
-    private List<JTest> tests;
+    private final String id;
+    private final List<JTest> tests;
 
-
-    public static Builder builder() {
-        return new Builder();
+    public static Builder builder(Id id, List<JTest> tests) {
```

Comment:
May we have a vararg version of the method? I.e. 
builder(Id id, JTest[]...jTests)

That will simplify code of jagger tests i.e. no need to use Arrays.asList().